src/api/cvc4cpp.h

```diff
@@ -590,6 +698,43 @@ class CVC4_PUBLIC Term
    */
   bool isNull() const;
 
+  /**
+   * @return true iff this Term has an operator
+   */
+  bool hasOperator() const;
+
+  /**
+   * @return true iff this Term has a builtin operator
```

Comment:
Can you change the format of the documentation of this and the function below? First description, then parameters (if any), then return value. And start description sentence with upper case.

Reply:
Nevermind, I see what you mean. Fixed with 6d9f9f2e531d824e57f6b0ac6e0f06d86abef3cb.